### ISSUE TITLE
Floodplain Accessibility

### ISSUE THREAD

**Issue by MB-Turnage** [Riverscapes/RiverscapesXML#520]
For the Floodplain Accessibility (non-raster) curation, what is the type of data here? According to the attribute table, the **FloodplainAccess** values do not appear to be binary as they are in the Floodplain Accessibility raster. Any information on what type of data this is (or if anything non 1 should be treated as zero, making it binary) , would be helpful here.

Thanks!
![flood_acc_attable](https://user-images.githubusercontent.com/127990296/226444149-db80e4f6-9958-496c-97b8-95622ca7ea42.PNG)

**Comment by jtgilbert** [comment]
These values are the proportion of the floodplain that is accessible (0-1) within a window associated with the line or point.

**Comment by jtgilbert** [comment]
@MB-Turnage typically for our continuous values we use ~5 bins. I like the color ramp, but maybe see how increasing it to 5 bins looks.

**Comment by MB-Turnage** [comment]
![fp_acc_snip3](https://user-images.githubusercontent.com/127990296/227325935-b3560a01-c182-48f5-a460-aa62a3bf0b3a.PNG)
![fp_acc_snip4](https://user-images.githubusercontent.com/127990296/227325986-bad83016-4768-4669-aa5a-e2f794fcdaa0.PNG)
![fp_acc_snip5](https://user-images.githubusercontent.com/127990296/227326019-3bedb752-21ba-41fe-911d-bf2de0b279da.PNG)
![fp_acc_snip6](https://user-images.githubusercontent.com/127990296/227326030-f65f97af-bca2-4bcd-ae7a-842f056928a2.PNG)

@jtgilbert Here is the symbology with 5 bins using the natural breaks (jenks). I included it red to blue and also yellow to blue in case you'd prefer a different color set.

**Comment by jtgilbert** [comment]
Awesome, looks good! I think I prefer the blue to red ramp. When you're splitting, though, you want to use either equal interval breaks or some pre-determined custom binning because the natural breaks will be different for every project, so you can have consistency across all projects of the same type.

**Comment by MB-Turnage** [comment]
![fp_acc_snip7](https://user-images.githubusercontent.com/127990296/227602950-80a40c38-b976-479a-b8e2-e6e07a2cb92f.PNG)

Alright that sounds good. Here is the newest version with 5 bins at regular intervals.

**Comment by Cashe93** [comment]
![image](https://github.com/Riverscapes/RiverscapesXML/assets/49880208/8240b657-516b-4e1f-b0fa-4a66e106cee0)
